proof/courier.go

```diff
@@ -543,21 +545,190 @@ type BackoffCfg struct {
 	MaxBackoff time.Duration `long:"maxbackoff" description:"The maximum backoff time to wait before retrying to deliver the proof to the receiver."`
 }
 
+// BackoffHandle is a handle to the backoff procedure.
+type BackoffHandle struct {
```

Comment:
nit: since this is a concrete type/struct, wouldn't it be a handle**r** (instead of just handle)? I initially thought this was a typo but it seems very consistent throughout the diff.